docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -36,7 +36,8 @@ The support timeframe for the SDK typically matches the VS it's included in.
 | 6.0.100-rc1      | 17.0-preview 4     | 16.10                               | September    | N/A       |
 | 6.0.100          | 17.0               | 16.10**                             | Nov. '21     |           |
 
-**Note:** The Minimum version is what's required to load the SDK but doesn't cover what's supported. Targeting net6.0 is officially supported in 17.0+ only.
+> [!NOTE]
+> The Minimum version is what's required to load the SDK but doesn't cover what's supported. Targeting net6.0 is officially supported in 17.0+ only.
 > \* MSbuild/VS supported for longer
```

Comment:
You'll need an empty line between the `>` to separate the note from the general `*` and `**` markers.